packages/app-project/src/screens/ProjectHomePage/components/Hero/helpers/fetchWorkflowsHelper/fetchWorkflowsHelper.js

```diff
@@ -0,0 +1,67 @@
+import { panoptes } from '@zooniverse/panoptes-js'
+import find from 'lodash/find'
+import pick from 'lodash/find'
+import toNumber from 'lodash/toNumber'
+
+function fetchWorkflowData (activeWorkflows) {
+  return panoptes
+    .get('/workflows', {
+      complete: false,
+      fields: 'completeness',
+      id: activeWorkflows.join(','),
+    })
+    .then(response => response.body.workflows)
+    .then(workflows => workflows.map(pickWorkflowFields))
+}
+
+function fetchDisplayNames (language, activeWorkflows) {
+  return panoptes
+    .get('/translations', {
+      fields: 'strings,translated_id',
+      language,
+      translated_id: activeWorkflows.join(','),
+      translated_type: 'workflow'
+    })
+    .then(response => response.body.translations)
+    .then(createDisplayNamesMap)
```

Comment:
Both of these requests will have to paginate when there's 20+ workflows (which there are projects that do this). I'm ok with this being updated along with the design for that in a separate PR.